cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -683,6 +806,17 @@ CALL {
 } IN TRANSACTIONS
 ----
 
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE (a {prop:7})-[r:R]->(b {prop: a.prop})
+----
+a|
+`CREATE` clauses where a variable introduced in the pattern is also referenced from the same pattern are deprecated.
```

Comment:
Same as previous suggestion

```suggestion
It is no longer allowed to have `CREATE` clauses in which a variable introduced in the pattern is also referenced from the same pattern.
```

Reply:
This is in the table for 4.4, where the feature was deprecated but still allowed, so it should say deprecated. But I can try to reformulate it slightly still